cms/static/sass/elements/_xblocks.scss

```diff
@@ -248,6 +248,18 @@
         color: $color-visibility-set;
       }
     }
+    
+    .action {
+        
+        .save {
+            @extend %btn-secondary-blue;
```

Comment:
As we discussed in Hipchat, make sure that:

- This doesn't negatively impact other XBlocks
- It has sufficient color contrast
- It looks good in the LMS (the change here is for CMS only). CMS and LMS should match.